frontend/src/pages/modelServing/screens/global/InferenceServiceTable.tsx

```diff
@@ -82,6 +84,24 @@ const InferenceServiceTable: React.FC<InferenceServiceTableProps> = ({
           setEditInferenceService(undefined);
         }}
       />
+      <ManageKServeModal
+        isOpen={!!editInferenceService && !isModelMesh(editInferenceService)}
+        editInfo={{
+          inferenceServiceEditInfo: editInferenceService,
+          servingRuntimeEditInfo: {
+            servingRuntime: servingRuntimes.find(
+              (sr) => sr.metadata.name === editInferenceService?.spec.predictor.model.runtime,
+            ),
+            secrets: [], // TODO: Get the secrets
```

Comment:
@christianvogt @DaoDaoNoCode doing a review i realized I didn't add the secrets, I need to do this later